js/utils/helpers.js

```diff
@@ -1871,21 +1871,21 @@
                                 } else if (attr[0] == "iframe-style") {
                                     iframeStyle = attr[1];
                                 } else if (attr[0] == "class") {
-                                    newClass += " " + attr[1];
+                                    newClass += (attr[1].length ? (attr[1] + " ") : "");
                                 } else {
                                     iframeHTML +=  attr[0] + '="' + attr[1] + '"';
                                 }
                                 iframeHTML += " ";
                             });
 
                             //During print we need to display that the iframe with YouTube video is replaced with a note
-                            if(isYTlink)
-                            {
-                              html += iframeHTML + newClass + "></iframe>";
-                              html = '<span class="video-info-in-print" style="visibility:hidden">' + videoText + "</span>" + html;
+                            if(isYTlink){
+                              html = '<span class="' + module._classNames.showInPrintMode + '" style="visibility:hidden">' + videoText + "</span>";
+                              newClass += module._classNames.hideInPrintMode;
                             }
-                            else
-                              html += iframeHTML + "></iframe>";
+                            // add closing quote
+                            newClass += '"';
+                            html += iframeHTML + newClass + "></iframe>";
```

Comment:
I have added the class attribute to the iframe as requested. I would like to point out that the class attribute was not added before to the iframe element and hence this change had to be incorporated for all iframes in the test case as well.